tldr/semantic.py

```
@@ -272,8 +272,10 @@ def extract_units_from_project(project_path: str, lang: str = "python", respect_
     project = Path(project_path).resolve()
     units = []
 
-    # Get code structure (L1)
-    structure = get_code_structure(str(project), language=lang)
+    from tldr.tldrignore import load_ignore_patterns
```

Ruff (0.14.10)

Redefinition of unused `load_ignore_patterns` from line 270

Remove definition: `load_ignore_patterns`

(F811)